entrypoint.sh

```diff
@@ -2,6 +2,9 @@
 
 if [ $1 == "validate" ]; then
     echo "validating helm charts"
+    if [ -n "$4" -a -n "$5" -a -n "$6" -a -n "$7" ]; then
```

Comment:
if we're removing docker from the flow, we don't need to bring the add in any more right?

Reply:
done.